### PR TITLE
Temp dir cleanup

### PR DESCRIPTION
**Pull request by cboesch-gpsw**

### CODE REVIEW

**Comment by mbouron-gpsw** [comment]
A separate commit should made get_temp_dir (cmp.py) private since it's only used locally. Also, shouldn't get_temp_dir (cmp.py) rely on get_nodegl_tempdir() ?

**Comment by ubitux** [comment]
> A separate commit should made get_temp_dir (cmp.py) private since it's only used locally. Also, shouldn't get_temp_dir (cmp.py) rely on get_nodegl_tempdir() ?

`get_temp_dir()` is removed/inlined using `get_nodegl_tempdir()` in "utils/tests: use get_nodegl_tempdir()". Do you want me to split that commit?

**Comment by mbouron-gpsw** [comment]
> > A separate commit should made get_temp_dir (cmp.py) private since it's only used locally. Also, shouldn't get_temp_dir (cmp.py) rely on get_nodegl_tempdir() ?
> 
> `get_temp_dir()` is removed/inlined using `get_nodegl_tempdir()` in "utils/tests: use get_nodegl_tempdir()". Do you want me to split that commit?

No, that's OK like this. LGTM then.

**Review by mbouron-gpsw — APPROVED** [review]

File: pynodegl-utils/pynodegl_utils/ui/graph_view.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def _init_ctx(self, rendering_backend):
         )
 
     def _update_graph(self, dot_scene):
-        basename = op.join(tempfile.gettempdir(), 'ngl_scene.')
+        basename = op.join(misc.get_nodegl_tempdir(), 'ngl_scene.')
```
Comment (mbouron-gpsw) [inline pynodegl-utils/pynodegl_utils/ui/graph_view.py:192]: You can remove the tempfile import.

Comment (cboesch-gpsw) [reply pynodegl-utils/pynodegl_utils/ui/graph_view.py:191]: Done

**Pull request merged**